src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -322,6 +322,17 @@ public async Task<HttpResponseMessage> CheckExportAsync(Uri contentLocation, Can
             return response;
         }
 
+        public async Task CancelExport(Uri contentLocation, CancellationToken cancellationToken = default)
+        {
+            var message = new HttpRequestMessage
```

Comment:
nit: `using var message = new HttpRequestMessage`

Reply:
Updated. Did not know that we could skip the braces for the `using` block starting from C# 8.0! Thanks :)